src/transformers/models/flaubert/modeling_tf_flaubert.py

```diff
@@ -862,12 +868,84 @@ def serving_output(self, output):
     """,
     FLAUBERT_START_DOCSTRING,
 )
-class TFFlaubertForSequenceClassification(TFXLMForSequenceClassification):
-    config_class = FlaubertConfig
-
+# Copied from transformers.models.xlm.modeling_tf_xlm.TFXLMForSequenceClassification with TFXLMForSequenceClassification->TFFlaubertForSequenceClassification, TFXLMPreTrainedModelTFFlaubertPreTrainedModel, TFXLMMainLayer->TFFlaubertMainLayer, XLM_INPUTS_DOCSTRING->FLAUBERT_INPUTS_DOCSTRING
```

Comment:
```suggestion
# Copied from transformers.models.xlm.modeling_tf_xlm.TFXLMForSequenceClassification with XLM_INPUTS->FLAUBERT_INPUTS,XLM->Flaubert
```
Should work as well (the replacements are done in order, so first the name of the docsting will be updated, then the others).